src/rust/engine/ui/src/console_ui.rs

```diff
@@ -118,11 +120,18 @@ impl ConsoleUI {
     // render might barely miss a data refresh.
     let draw_target = ProgressDrawTarget::term(term, Self::render_rate_hz() * 2);
     let multi_progress = MultiProgress::with_draw_target(draw_target);
-
-    let bars = (0..self.local_parallelism)
+    multi_progress.set_move_cursor(false);
+    let terminal_dimensions = terminal_size().unwrap_or((
```

Comment:
It'd be nice to use `set_move_cursor(false)` because it eliminates flickering, but I can't seem to figure out how to get `indicatif` to "wipe" the lines with no message, or to not leave artifacts when the command finishes running.

I tried in the `pbar.set_message` call in the loop to set the message to a bunch of spaces (to clear the msg) but that didn't work.